### PR TITLE
Add compatibility with PHP 8.2

### LINKED ISSUE #42: `laminas/laminas-cache` is specified in both `require` AND `require-dev` sections of `composer.json`
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | 2.1.0

#### Summary

Reviewing these cache packages, I noticed the titular problem which I was previously told explicitly to NOT do:
- https://github.com/composer/composer/issues/3742#issuecomment-74540885.

#### Current behavior

`composer install --no-dev` installs `laminas/cache` `3.0.x-dev` and not `3.0`. Whereas this may suffice for the use case here as neither the lock file nor `require-dev` section SHOULD be read during installation as a library, this isn't guaranteed as noted om the above issue.

#### How to reproduce

Check out repository locally and run `composer install --no-dev`.

#### Expected behavior

A package should appear only in `require` OR `require-dev` and not both.

### PR DESCRIPTION
**Pull request by boesing**
|    Q          |   A
|-------------- | ------
| New Feature   | yes

### Description

This also removes support for PHP 7.4
Closes #42

### CODE REVIEW

File: .laminas-ci/install-apcu-extension-via-pecl.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PHP_VERSION="$1"
+
+if ! [[ "${PHP_VERSION}" =~ 8\.2 ]]; then
+  echo "APCu is only installed from pecl for PHP 8.2, ${PHP_VERSION} detected."
+  exit 0;
+fi
+
+set +e
+apt install make
```
Comment (boesing) [inline .laminas-ci/install-apcu-extension-via-pecl.sh]: Could be removed if we re-add `make` to the container.

Ref: https://github.com/laminas/laminas-continuous-integration-action/issues/135

Comment (Ocramius) [reply .laminas-ci/install-apcu-extension-via-pecl.sh]: Note that you may want a `-y` on these commands anyway :D

Comment (Ocramius) [reply .laminas-ci/install-apcu-extension-via-pecl.sh]: I'll release a new CI action, and get rid of this line :+1:

**Review by Ocramius — APPROVED** [review]
:ship: 

Just beware: @boesing's commits were signed, and my rebase corrupted the signature (expected)

File: .laminas-ci/install-apcu-extension-via-pecl.sh (hunk 1 above)
Comment (Ocramius) [reply .laminas-ci/install-apcu-extension-via-pecl.sh]: 024c7da3a0b1f6e057d43bcfaeae730ff4e4d882

**Pull request merged**